test/bouture.test.js

```diff
@@ -308,3 +308,19 @@ describe(`Bouture.button({onceClick: e => { state *= 2 }})`, () => {
     expect(state).to.equal(37)
   })
 })
+
+describe(`Bouture('custom-element', Bouture('paper-badge'))`, () => {
+  it('should create a element', () => {
+    const customElt = Bouture('custom-element', Bouture('paper-badge', {label: '3'}))
+
+    expect(Object(customElt)).to.equal(customElt)
+  })
+})
+
+describe(`Bouture('custom-element', Bouture('paper-badge')).getElement()`, () => {
+  it('should return an actual DOM node', () => {
+    const customElt = Bouture('custom-element', Bouture('paper-badge', {label: '3'})).getElement()
+
+    expect(customElt).to.be.an.instanceof(HTMLElement)
```

Comment:
This assertion is good but insufficient in my opinion. For instance, it doesn't test that the element created belongs to the correct interface.

See the `<word-count>` example from [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry/define#Examples). A `WordCount` class is created.
I think `Bouture('paper-badge')` should be an instance of `PaperBadge`. This would require loading the Polymer element for the test, etc.